components/constellation/Cargo.toml

```diff
@@ -45,5 +45,5 @@ servo_url = {path = "../url"}
 webvr_traits = {path = "../webvr_traits"}
 webrender_api = {git = "https://github.com/servo/webrender", features = ["ipc"]}
 
-[target.'cfg(all(not(target_os = "windows"), not(target_os = "ios")))'.dependencies]
+[target.'cfg(all(not(target_os = "windows"), not(target_os = "ios"), not(target_os="android"), not(target_arch="arm"), not(target_arch="aarch64")))'.dependencies]
```

Comment:
I wonder if we should just do `any(target_os = "macos", target_os = "linux")` at this point :/

Reply:
Well Linux on arm is still Linux, right ?

Reply:
Good point!